tests/jsTests/validationSpec.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Suite : validateNumber
+ * Description : Test suite to validate comrades'phone number
+ */
+describe('validateNumber', function() {
+    it('Should be phone number', function() {
+        expect(validation.isPhoneNumber('342345235')).toBeTruthy();
+    });
+
+    it('Should not be phone number', function() {
+        expect(validation.isPhoneNumber('324fdgfd215235')).not.toBeTruthy();
+    });
+});
+
+/**
+ * Suite : validateEmail
+ * Description : Test suite to validate users' email address
+ */
+describe('validateEmail', function() {
+    it('Should be email', function() {
+        expect(validation.isEmailAddress('abc@def.com')).toBeTruthy();
+    });
+
+	it('Should be email', function() {
+        expect(validation.isEmailAddress('mail.google.com')).not.toBeTruthy();

```

Comment:
@MehaKaushik This was just a basic test as an expectation for learning JS testing. The expectation was to add one test for any one method for any on the files to illustrate learning. As suggested by @Anjali05, More tests will be added at a later stage, I am still adding more dependencies and jasmine jquery libraries for testing.